src/msg/async/Event.h

```diff
@@ -85,6 +86,7 @@ class EventDriver {
  * EventCenter maintain a set of file descriptor and handle registered events.
  */
 class EventCenter {
+  using clock_type = ceph::coarse_mono_clock;
```

Comment:
thought you wanted to use `coarse_real_clock`... so we can update the commit message accordingly?